### DISCUSSION TITLE
Prowlarr Provider Icons non-individualized.

### DISCUSSION THREAD

**Discussion by neo-neo1** [pymedusa/Medusa#9893] [Ideas & Feature requests]
When using several providers via Prowlarr during a manual search page the "Provider" table uses all identical nondescript Prowlarr icons for different providers. This is confusing since glancing at individualized provider icons is a quick and efficient way to differentiate which provider the result corresponds to, on the manual search page.

To better illustrate the issue refer to the below screenshots:

Using [Medusa configured providers](https://i.imgur.com/D8Lvano.jpg) and using [Prowlarr configured providers](https://i.imgur.com/MyOKdoK.jpg) on the manual search page.

Using identical icons for Prowlarr indexers would be like having identical icons for indexers configured directly under Medusa. Please use individualized provider icons for providers configured under Prowlarr. 

**Optionally** perhaps modify the provider icons configured under Prowlarr with a different tint/hue or add a P over it. Something to differentiate it besides them all being identical duplicates.

P.S. Yes you can hover the mouse over the icon, which becomes needlessly tiresome and adds time when going down a long list of results. Using a mouse to hover over duplicate icons should not be the default use case.

**Comment by p0psicles** [comment]
It's on my to do. But not that easy to implement. But was thinking of keeping a looking mapping of prowlarr providers that we also have in Medusa. And else show prowlarr icon. As we don't get icons returned from prowlarr

**Reply by neo-neo1** [reply]
> It's on my to do. But not that easy to implement. But was thinking of keeping a looking mapping of prowlarr providers that we also have in Medusa. And else show prowlarr icon. As we don't get icons returned from prowlarr

Could have a user input field for icon URLs regarding providers that Medusa does not have.  Or point to a local folder of icons.

**Comment by Rouzax** [comment]
@p0psicles 
Would another option just be to show the Provider Name instead of the icon?
If that was an option that would already help me greatly, more and more providers are going through Prowlarr for me making manual searches difficult.
